apis/io-engine/protobuf/v1/stats.proto

```diff
@@ -8,18 +8,27 @@ package mayastor.v1;
 service StatsRpc {
   // RPC to get Pool IO stats.
   rpc GetPoolIoStats (ListStatsOption) returns (PoolIoStatsResponse);
+  // RPC to get Nexus IO stats.
+  rpc GetNexusIoStats (ListStatsOption) returns (NexusIoStatsResponse);
```

Comment:
Should we add a uuid to the IoStats? Then we can expose both uuid and name of the nexus - same for pool actually

Reply:
Okay ill add